src/o365/spo/SpoAuth.ts

```diff
@@ -11,6 +11,11 @@ interface Token {
   expiresOn: string;
 }
 
+export interface FormDigest {
```

Comment:
Why is this interface added to the `SpoAuth` file?

Reply:
It felt like it belonged in SpoAuth rather then the SpoCommand as Digest is Auth related.  Do you want it moved to the SpoCommand?

Reply:
In our case we use auth primarily for communication with Azure AD, and FormDigest is not required for all commands, so I'd suggest moving it to `SpoCommand`